src/views/exercices_root.php

```diff
@@ -0,0 +1,43 @@
+<header class="heading answering">
+  <section class="container">
+    <a href="/"><img src="public/assets/img/logo.png" /></a>
```

Comment:
sorry for misunderstood but the file logo should be in the path public/assets/img/logo.png but here you should put /assets/img/logo.png